packages/fxa-auth-server/fxa-oauth-server/config/dev.json

```diff
@@ -188,14 +188,20 @@
     "enabled": true,
     "enabledClientIds": [
       "98e6508e88680e1a",
-      "dcdb5ae7add825d2"
+      "dcdb5ae7add825d2",
+      "ea3ca969f8c6bb0d"
     ]
   },
   "openid": {
     "issuer": "http://127.0.0.1:3030",
     "keyFile": "../config/key.json",
     "key": {}
   },
+  "ppid": {
+    "enabled": true,
+    "enabledClientIds": ["ea3ca969f8c6bb0d"],
+    "rotatingClientIds": ["ea3ca969f8c6bb0d"]
```

Comment:
I'm not really happy with this either, this was only done to make the [oauth_tests.js](https://github.com/mozilla/fxa/blob/b91441baa744e27e9e1e89049f61f8f8a503ca55/packages/fxa-auth-server/test/remote/oauth_tests.js#L263) module pass, I would have asssumed the test OAuth server would have started using test.json, but it's started using dev.json. When I force the NODE_ENV environment variable to test for the server, then I caused other errors. Still may be something that's straight forward to figure out.